src/backend/db/products.js

```diff
@@ -8,23 +10,50 @@ import { v4 as uuid } from "uuid";
 export const products = [
   {
     _id: uuid(),
-    title: "You Can WIN",
-    author: "Shiv Khera",
-    price: "5000",
-    categoryName: "non-fiction",
+    image: saree1,
+    title: "Sapphire Banarasi Saree",
+    // author: "Shiv Khera",
+    price: "Rs. 3,990 only",
+    categoryName: "women-wear",
+  },
+  {
+    _id: uuid(),
+    image: saree2,
+    title: "Banarasi Silk Saree",
+    // author: "Junaid Qureshi",
+    price: "Rs. 3,390 only",
+    categoryName: "women-wear",
+  },
+  {
+    _id: uuid(),
+    image: saree3,
+    title: "Black Banarasi Saree",
+    // author: "Shiv Khera",
+    price: "Rs. 3,340 only",
+    categoryName: "women-wear",
+  },
+  {
+    _id: uuid(),
+    image: saree4,
+    title: "Purple BanarasiSaree",
+    // author: "Shiv Khera",
+    price: "Rs. 3,590 only",
+    categoryName: "women-wear",
   },
   {
     _id: uuid(),
-    title: "You are Winner",
-    author: "Junaid Qureshi",
-    price: "3000",
-    categoryName: "horror",
+    image: saree5,
+    title: "Red Banarasi Saree",
+    // author: "Shiv Khera",
+    price: "Rs. 3,560 only",
+    categoryName: "women-wear",
   },
   {
     _id: uuid(),
-    title: "Think and Grow Rich",
-    author: "Shiv Khera",
-    price: "1000",
-    categoryName: "fiction",
+    image: saree6,
+    title: "Maroom Banarasi Saree",
+    // author: "Shiv Khera",
```

Comment:
In case you forgot just remove the commented code

Reply:
done